RELEASING.adoc

```diff
@@ -1,15 +1,27 @@
 = Releasing Kiali
 
-Kiali is released using a Jenkins Pipeline. If you need to release
-a new version of Kiali, please ask to a member of the QE team to
-trigger the release for you. Alternatively, ask the QE team for
-access to the Jenkins instance and trigger the release by yourself.
-
-Currently, the Pipeline supports only _minor_ and _snapshot (end-of-week)_
-releases and will release the back-end and the front-end together. If
-you need to do a patch or a major release, you'll need to
+Kiali is released using a Jenkins Pipeline, which is scheduled to
+run each Friday between 2pm and 3pm of America/Merida timezone.
+Every three Fridays a minor release is built. The other Fridays,
```

Comment:
`We build a release weekly, on Fridays: a minor release every three weeks, and snapshots on the other weeks.`

Maybe something like this would be better?